interface/controller/app_controller.go

```diff
@@ -0,0 +1,107 @@
+package controller
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/lscantillowl/academy-go-q42021/domain/model"
+)
+
+// respondWithJSON write json response format to the response writer
+func respondWithJSON(w http.ResponseWriter, code int, payload interface{}) {
+	response, err := json.Marshal(payload)
+	if err != nil {
+		log.Fatalf("Can not convert payload to JSON - %s", err)
+		w.WriteHeader(http.StatusInternalServerError)
+	}
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(code)
+	w.Write(response)
+}
+
+// Home handler to test the api
+func HomeHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
as a good practice of clean architecture, this handler should exist on its own controller.go file although it could be part of package controller